charts/k8s-monitoring/templates/tests/test.yaml

```diff
@@ -75,57 +75,39 @@ metadata:
     {{- end}}
   annotations:
     "helm.sh/hook": test
-    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-delete-policy": before-hook-creation
```

Comment:
Is any cleanup required by the user after running helm test?

Reply:
They just... linger there. I don't love it.
I'm not sure I'm happy with this PR as it stands.